mobx/test/exceptions_test.dart

```diff
@@ -26,4 +26,15 @@ void main() {
       expect(ex.stackTrace, isNotNull);
     }
   });
+
+  test('should preserve stacktrace', () async {
+    final stackTrace = StackTrace.current;
+    try {
+      Computed(() => Error.throwWithStackTrace(Exception('test'), stackTrace))
```

Comment:
Can we add a test with a simple exception instead of using `Error.throwWithStackTrace` ? Not all code will use this technique. We should see if the stacktrace is picked up automatically from deeper exceptions in code

Reply:
done